be/src/util/s3_util.cpp

```diff
@@ -269,7 +270,7 @@ std::shared_ptr<Aws::Auth::AWSCredentialsProvider> S3ClientFactory::get_aws_cred
 
     if (s3_conf.cred_provider_type == CredProviderType::InstanceProfile) {
         if (s3_conf.role_arn.empty()) {
-            return std::make_shared<Aws::Auth::InstanceProfileCredentialsProvider>();
+            return std::make_shared<Aws::Auth::DefaultAWSCredentialsProviderChain>();
```

Comment:
we should not touch the existing impl which is going to be removed in the future?